config/slash_commands/esv.rb

```diff
@@ -0,0 +1,39 @@
+Houston::Slack.config do
+  slash("esv") do |e|
+    if e.text.blank?
+      # Print usage
+      e.respond! "No search query given. Usage: `/esv <reference>` where reference is a verse reference, e.g. John 3:16 (book abbreviations _are_ supported)."
+    else
+      connection = Faraday.new(url: "http://www.esvapi.org")
+      query = "/v2/rest/passageQuery?#{build_query_with_reference(e.text)}"
+      response = connection.get(query)
+
+      if response.status != 200
+        e.respond! "It looks like ESV is not available at the moment. :sweat:"
+      else
+        e.respond! style_passage(response.body)
```

Comment:
`e.respond!` responds so that only the user who issued the command can see the response.

Do you think the response should be posted in the channel so others can see or not? (That would be `e.reply`, I think)